src/option.c

```diff
@@ -537,3 +539,83 @@ coap_option_filter_get(const coap_opt_filter_t filter, uint16_t type) {
    * but as _set and _unset do, the function does not take a const). */
   return coap_option_filter_op((uint16_t *)filter, type, FILTER_GET);
 }
+
+coap_optlist_t *coap_new_optlist(uint16_t number,
```

Comment:
Please put a newline after the return data type as mandated by the libcoap coding style guidelines.

Reply:
Fixed in Update